dashboard/src/components/hackathon/HackathonTeamMember.vue

```diff
@@ -171,6 +177,11 @@ const removeTeamMember = (member) => {
       }
     },
     onSuccess() {
+      if (member.email == session.user){
+        router.push(`/hack/${props.hackathon.permalink}`)
+        toast.warn('You have left the team.')
```

Comment:
Typo in function name - warn -> warning

Raises error: 
<img width="972" alt="Screenshot 2024-07-15 at 7 24 39 PM" src="https://github.com/user-attachments/assets/32b206f8-76db-4bc7-8fb8-e87b179212f8">